config/prow/plugins.yaml

```diff
@@ -117,6 +117,23 @@ label:
     # This label is used by the API review process
     # https://git.k8s.io/community/sig-architecture/api-review-process.md#mechanics
     - api-review
+    # These labels are used by Kubeflow
+    # TODO(https://github.com/kubernetes/test-infra/issues/8648): Switch
+    # to configuring prefixes and not individual labels once  that's supported.
+    - community/discussion
+    - community/maintenance
+    - community/question
+    - cuj/build-train-deploy
+    - cuj/multi-user
+    - platform/aws
+    - platform/azure
+    - platform/gcp
+    - platform/minikube
+    - platform/other
+    - tide/merge-method-merge
```

Comment:
I think only the tide labels from here down are used outside kubeflow.

Labels like `platform/*` would be `area/foo` in a kubernetes project. `community/foo` would be `kind/foo`. the `cuj/*` labels seem very specific to kuebflow.

Reply:
Thats correct, I found the PR from the blame: https://github.com/kubernetes/test-infra/pull/10515

Reply:
Annnnd its already been fixed: https://github.com/kubernetes/test-infra/pull/21131 :man_facepalming: 
Sorry for the noise.